AUTHORSHIP.md

```diff
@@ -0,0 +1,71 @@
+# Authorship guidelines for academic papers and software archives
+
+First of all, we are deeply thankful to everyone who has helped make Fatiando a
+Terra what it is today. Our goal for this document is to establish guidelines
+for giving credit to contributors for their work.
+To do so, we will attempt to define:
+
+- Fair and diverse ways of providing recognition for contributors' efforts.
+- Define _contributions_ in a broad way: writing code and/or documentation,
+  providing ideas, fostering the community, etc.
+
+The following are the ways in which individuals who have contributed will be
+recognized.
+
+> **Note**: These policies are not set in stone and may be changed to
+> accommodate the growth of the project or the preferences of the community.
+
+## The `AUTHORS.md` file
+
+Anyone who has contributed a pull request to the project is welcome to add
+themselves to the `AUTHORS.md` file. This file lives in the repository and is
+packaged with distributions. This is an optional process.
+
+## Changelog for each release
+
+Every time we make a release, everyone who has made a commit to the repository
+since the previous release will be mentioned in the changelog entry. If their
+full name is available on GitHub, we will use it. Otherwise, we will use the
+GitHub handle. This is a way of saying "Thank you".
+
+## Authorship on Zenodo archives of releases
+
+Anyone who has contributed to the repository (i.e., is know to `git log`) will
```

Comment:
Should `(i.e., is know to `git log`)` be `known to `git log`` or even `documented by `git log``?

Reply:
Thanks for the suggestion @jessepisel. I changed to "appears on `git log`"